masterlist.yaml

```diff
@@ -976,6 +1017,21 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5941/' ]
     after:
       - 'Relationship Dialogue Overhaul.esp'
+  - name: 'XPMSE.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/1988/'
+        name: 'XP32 Maximum Skeleton Special Extended - XPMSSE on Nexus Mods'
+    msg:
+      - <<: *doNotClean
+      - <<: *requireSKYUI
+        subs: [ '[XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/)' ]
+        condition: 'file("scripts/XPMSEMCM.pex") and not active("SkyUI_SE.esp")'
```

Comment:
Due to the set up of this mods Fomod there were several scenarios where SkyUI would be required.
As this script was always present when SkyUI was required, it seemed like the easiest way to check.